bokeh/resources.py

```diff
@@ -303,13 +303,13 @@ def _resolve(self, kind):
         return (files, raw)
 
     def _inline(self, path):
-        begin = "/* BEGIN %s */" % path
+        begin = "/* BEGIN %s */"
```

Comment:
You should remove `%s` as well (the same below). Preferably use `os.path.basename(path)`, so that we know what component we're dealing with.